doc/tor.md

```diff
@@ -211,3 +256,11 @@ for normal IPv4/IPv6 communication, use:
   Otherwise it is trivial to link them, which may reduce privacy. Onion
   services created automatically (as in section 2) always have only one port
   open.
+
+- If you are in an environment that does not permit direct Tor connections,
+  and does not permit the public bridges, then considering the trade-offs
+  it probably isn't safe enough to use the Tor network. Example: few places
+  in China and India
```

Comment:
It may be more maintainable to not cite specific countries or regions, as situations can change.

Writing suggestion:

- If you are in an environment that does not permit direct Tor connections or public bridges, it may not be safe enough to use the Tor network.

Reply:
who cares?
Doc's are changing too, and those two county's deserve to be called out now. imho.
I am glad to put this PR on my alert and TODO list and hail them for change instantly.

I know young devs ( not u ) have problems with dev or promoting Tor and calling out autocratic states and none of those devs must feel tempted to ACK here, but I had my life and can do this and give a shit if that is polite or not, if it is a fact.

Reply:
I had mentioned those two countries in this Q&A on Stackexchange: https://bitcoin.stackexchange.com/questions/98772/what-are-the-safe-ways-to-connect-to-bitcoin-network-using-tor

I found few things mentioned in an article about Tor issues and since I am from India I was aware of some incidents here.

One example from China: http://www.asianews.it/news-en/Beijing-arrests-engineer-who-tried-to-bypass-Internet-censorship-32691.html

One example from India: https://www.newslaundry.com/2020/03/05/kashmiris-use-vpn-to-skirt-internet-curbs-but-is-vpn-really-secure

Censorship in some parts of India is not only limited to Tor, VPN, speed of Internet etc. but a lot of times Internet is completely shutdown in last few years for days in some places: https://internetshutdowns.in/ (_Text messages work during such times so I had also worked on a side project to broadcast bitcoin transaction without Internet: https://github.com/prayank23/OfflineTx but thats not related to this PR_)

Reply:
It's not a 'few places' in China and India; it is China and India as a whole. As you mentioned, you were aware of an incident. Others in your country may not be aware. It is better to include some warning in this doc to what we can here to prevent another incident. The part of the text does need a little work.

**From:** 
> If you are in an environment that does not permit direct Tor connections,
  and does not permit the public bridges, then considering the trade-offs
  it probably isn't safe enough to use the Tor network. Example: few places
  in China and India

**To:**

`Some states have the ability to monitor and block use of the Tor network. Recent examples include China and India. If you are in an environment that does not permit direct Tor connections or the use of Tor bridges, then considering the trade-offs, it may not be safe to use the Tor network.
`

Reply:
Removed China and India.